### PR TITLE
Add styleguide entry for dialog component

### PR DESCRIPTION
**Pull request by pcardune**
@breville your wish is my command:
screen shot 2016-06-20 at 4 59 36 pm

### CODE REVIEW

**Comment by islemaster** [comment]
LGTM.

File: apps/src/templates/StyleGuide.jsx [hunk 1]
```diff
@@ -59,11 +59,11 @@ function Colors() {
...
       <p>
         JavaScript usage is as follows:
-        <pre>
-          {`import colors from '@cdo/apps/colors';
-console.log("the hex value for purple is:", colors.purple);`}
-        </pre>
       </p>
+      <pre>
+        {`import colors from '@cdo/apps/colors';
+console.log("the hex value for purple is:", colors.purple);`}
```
Comment (breville) [inline apps/src/templates/StyleGuide.jsx:65]: inadvertent console.log?

**Comment by breville** [comment]
lgtm

File: apps/src/templates/StyleGuide.jsx (hunk 1 above)
Comment (pcardune) [inline apps/src/templates/StyleGuide.jsx:65]: nope. This shows up in the style guide as example code. It doesn't actually run.

**Pull request merged**